### PR TITLE
Updating whatsnew for the week

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) updates whatsnew since February 24th.

## Affected DevDocs pages

-  https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,45 @@ description: |
...
+- description: 'Added release notes for the following Magento Cloud Suite packages:  [ece-tools
+    v2002.0.23](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html),<br/>[Magento
+    Cloud Patches v1.0.2](https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html),
+    and [Magento Cloud Components v1.0.3](https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html).'
+  versions: 2.x
+  type: Technical changes
+  date: February 27, 2020
+  link: https://github.com/magento/devdocs/pull/6700
+- description: Added topic for [GridDataProvider UI component](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-grid-data-provider.html).
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml]: missing version

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,45 @@ description: |
...
+  type: New topic
+  date: February 26, 2020
+  link: https://github.com/magento/devdocs/pull/6728
+- description: Added topic for the [ColumnsEditorRecord](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-columns-editor-view.html)
+    UI component.
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml]: missing version

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,45 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Feb 24 12:34:55 2020
+updated: Wed Mar 4 10:30:29 2020
 entries:
+- description: Updated the [Magento Cloud reference topic](https://devdocs.magento.com/cloud/reference/cli-ref-topic.html)
+    in the Cloud guide.
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml]: missing version

**Comment by erikmarr** [comment]
running tests

**Pull request merged**